python/whylogs/api/writer/whylabs.py

```diff
@@ -115,8 +115,8 @@ class WhyLabsWriter(Writer):
     A WhyLogs writer to upload DatasetProfileView's onto the WhyLabs platform.
 
     >**IMPORTANT**: In order to correctly send your profiles over, make sure you have
-    the following environment variables set: `[WHYLABS_ORG_ID, WHYLABS_API_KEY, WHYLABS_DEFAULT_DATASET_ID]`. You
-    can also follow the authentication instructions for the why.init() method at {_INIT_DOCS}.
+    the following environment variables set: `[WHYLABS_ORG_ID, WHYLABS_API_KEY, WHYLABSdefault_DATASET_ID]`. You
```

Comment:
```suggestion
    the following environment variables set: `[WHYLABS_ORG_ID, WHYLABS_API_KEY, WHYLABS_DEFAULT_DATASET_ID]`. You
```

Reply:
Woops. Can you tell I refactored something by doing a search/replace for `_default`...